calico_containers/calicoctl.py

```diff
@@ -656,74 +657,15 @@ def node_show(detailed):
     print str(x) + "\n"
 
 
-def save_diags():
+def save_diags(upload=False):
```

Comment:
I don't think you need this =False. If the flag isn't supplied then it will already be false.